packages/components/src/search-control/README.md

```diff
@@ -23,6 +24,7 @@ function MySearchControl( { className, setState } ) {
 
     return (
         <SearchControl
+			label={ __( 'Search posts' ) }
             value={ searchInput }
```

Comment:
```suggestion
            label={ __( 'Search posts' ) }
            value={ searchInput }
```

I'm not sure whether the existing lines should be updated to tabs or if the new line should be switched to spaces but either way might be nice to fix the layout of the example.